rules/brand-more-than-logo/rule.md

```diff
@@ -0,0 +1,74 @@
+<script async src="//www.instagram.com/embed.js"></script>
+**Video: Your brand is more than just a logo (40 sec)**
+
+## Key elements of brand identity
+
+### 1. Color Palette: The Emotional Trigger
+Your brand colors evoke specific emotions. Consistent use helps build recognition and trust. Examples include:
+- **Blue:** Trust and stability (e.g., IBM, Visa).
+- **Red:** Energy and excitement (e.g., Coca-Cola, Netflix).
+- **Green:** Growth and tranquility (e.g., Starbucks, Whole Foods).
+
+### 2. Typography: Communicating Personality
+Fonts do more than look good—they communicate your brand’s personality.
+- **Sans-serif fonts** (e.g., Helvetica): Modern and approachable.  
+- **Serif fonts** (e.g., Times New Roman): Traditional and reliable.  
+- **Custom fonts** (e.g., Coca-Cola’s script): Unique to the brand.
+
+### 3. Tone of Voice: Shaping Perception
+How you say things matters just as much as what you say. Examples of tone in action:
+- **Formal:** Financial institutions (e.g., Goldman Sachs).
+- **Conversational:** Consumer apps (e.g., Slack).
+- **Inspirational:** Athletic brands (e.g., Nike).
+
+## Brand guidelines ensure consistency
+
+Without guidelines, brand communication becomes inconsistent. A brand guideline includes:
+- **Color usage** (HEX, RGB values)
+- **Typography standards** (font sizes, spacing)
+- **Logo rules** (placement, acceptable variations)
+- **Imagery guidelines** (photo styles and iconography)
+- **Tone of voice examples** (proper messaging across platforms)
+
+::: info
+#### Case Study: Google's Seamless Brand Identity
+Even without their logo, Google’s identity is recognizable through their playful tone, minimalistic design, and consistent color use. This demonstrates the power of cohesive branding across all touchpoints.
+:::
+
+## A logo is only the beginning
+
+Your logo may be the face of your brand, but it is not the full story. By aligning all brand elements—colors, typography, messaging, and voice—you build a strong, lasting impression that resonates with customers. **Consistency** across every interaction fosters trust and makes your brand instantly recognizable.
+
+**Key Takeaways:**
+- **Consistency matters:** Uniform use of brand elements builds trust.
+- **Create emotional connections:** Align brand components with your values to foster relationships.
+- **Use visual identity wisely:** Make your visuals reflect your brand’s message.
+- **Evolve over time:** Review your brand strategy to keep it relevant.
+
+::: info
+**Pro Tip:** Regularly review your brand guidelines to ensure they align with your company’s growth and market changes.
```

Comment:
@bettybondoc 

- [ ] 1. Please make the word "tip" lowercase
- [ ] 2. Can we add a link with an example for brand guidelines? E.g. SSW

Reply:
@tiagov8 I don't think we can use ours as an example just yet. Maybe once we make the new branding pages?

https://www.ssw.com.au/company/typography

Reply:
@bettybondoc 

Maybe link to https://www.ibm.com/design/language/

Reply:
@tiagov8 Done - link added :)